### PR TITLE
Api/Exclude wrong words

### PR DESCRIPTION
**Pull request by nienna73**
## What's in this PR:
* exclude recordings with the wrong word
* exclude recordings with the wrong speaker

**Notes:**
If we're comfy merging this into development, I'll manually merge it into **production** at the same time.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -12,4 +12,4 @@ services:
         - "/data/application-data/speech-db/audio:/app/data/audio"
         - "/data/maskwacis-recordings/sessions:/app/data/sessions"
         - "/data/maskwacis-recordings:/data/maskwacis-recordings"
-        - "/data/application-data/speech-db/metadata.csv:/app/private/metadata.csv"
+        - "/data/application-data/speech-db/metadata.csv:/app/private/metadata.csv"
```
Comment (andrewdotn) [inline docker-compose.yml:15]: I expect you didn’t intend to remove the trailing newline here? PyCharm → Preferences → Editor → General → scroll *waayyy* down on the right → “Ensure every saved file ends with a line break”

**Review by eddieantonio — COMMENTED** [review]
Could you please update the tests?

https://github.com/UAlbertaALTLab/recording-validation-interface/blob/development/validation/tests/test_recordings_search.py#L239-L273

File: docker-compose.yml (hunk 1 above)
Comment (nienna73) [reply docker-compose.yml:15]: I added a line to this file and pushed it straight to Production, but didn't need that line in this PR, so I removed it and it took the newline with it 🤦 
Good to know PyCharm can stop this from happening though!

**Comment by nienna73** [comment]
How could I forget the tests!? On it.

**Review by eddieantonio — APPROVED** [review]
LGTM! Feel free to merge this into production! :shipit:

**Pull request merged**